### DISCUSSION TITLE
How to show line index in the first column in listbox?

### DISCUSSION THREAD

**Discussion by jim-jiang-github** [AvaloniaUI/Avalonia#12034] [Q&A]
Like this:
![image](https://github.com/AvaloniaUI/Avalonia/assets/29189410/0ebc171c-31e9-4e0d-9174-bdac4bf3896f)

Does anyone have an example?

**Comment by maxkatz6** [comment]
You should use AvaloniaEdit control as it has text virtualization and line indexes build in.
